bigbluebutton-html5/imports/startup/server/userPermissions.js

```diff
@@ -128,17 +128,14 @@ export function isAllowedTo(action, credentials) {
     userId,
   });
 
-  const allowedToInitiateRequest =
-    user &&
+  const allowedToInitiateRequest = user &&
     user.authToken === authToken &&
     user.validated &&
     user.clientType === 'HTML5' &&
     user.user &&
     user.user.connection_status === 'online';
 
-  const listOfSafeActions = [
-    'logoutSelf',
-  ];
+  const listOfSafeActions = new Array('logoutSelf');
```

Comment:
For this situation it will work, there are 2 other places where we do this same type of thing and doing it on a single line causes the +100 characters on a line error. In those cases we have to user multiple lines. I just got that working though without the new operator.